app/serializers/conditional_logic_form/form_item_serializer.rb

```diff
@@ -3,7 +3,7 @@
 module ConditionalLogicForm
   # Serializes Questioning or QingGroup for creating display and skip rule form fields in browser
   class FormItemSerializer < ApplicationSerializer
-    attributes :id, :display_if, :code, :rank, :full_dotted_rank, :form_id, :type
+    attributes :id, :display_if, :code, :rank, :full_dotted_rank, :form_id, :type, :group_name_translations
```

Comment:
@sdroadie any ideas why this is here? I can't remember a reason for this to be here. Perhaps we can remove?

Reply:
Ah, yeah, I forgot I added this. Totally not needed.